datasketches-memory-java8/src/main/java/org/apache/datasketches/memory/Buffer.java

```diff
@@ -57,7 +57,9 @@ static Buffer wrap(ByteBuffer byteBuffer) {
    * @param byteOrder the ByteOrder to be used. It must be non-null.
    * @return a new <i>Buffer</i> for read-only operations on the given ByteBuffer.
    */
-  static Buffer wrap(ByteBuffer byteBuffer, ByteOrder byteOrder) {
```

Comment:
Formatting:  For all the public classes under ds/memory/ and for methods with more than one parameter, I have each parameter on a separate line.  This is not only more readable, but comparing APIs across similar classes is much easier too.